src/static/css/content.less

```diff
@@ -602,11 +641,11 @@ p.print-georgia-para {
   li:first-child {
     text-align: center;
     border-left: none;
-    line-height: 75px;
+    line-height: 50px;
```

Comment:
This whole repo-list pattern is so janky. We should redo it with Flexbox or Grid or something.

Reply:
See previous comment.